allennlp/common/tee_logger.py

```diff
@@ -61,3 +61,7 @@ def isatty(self):
     def cleanup(self) -> TextIO:
         self.log.close()
         return self.terminal
+
+    @staticmethod
+    def getvalue() -> str:
```

Comment:
This is just something I did so some tests not fail in PyCharm.

Reply:
What failed?

Reply:
PyCharm has its own ways of running the tests, and it seems that it supposes there exists a `getvalue` method in `sys.stdout`:

```
Traceback (most recent call last):
  File "/snap/pycharm-professional/167/helpers/pycharm/_jb_unittest_runner.py", line 35, in <module>
    main(argv=args, module=None, testRunner=unittestpy.TeamcityTestRunner, buffer=not JB_DISABLE_BUFFERING)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/main.py", line 95, in __init__
    self.runTests()
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/main.py", line 256, in runTests
    self.result = testRunner.run(self.test)
  File "/snap/pycharm-professional/167/helpers/pycharm/teamcity/unittestpy.py", line 308, in run
    return super(TeamcityTestRunner, self).run(test)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/runner.py", line 176, in run
    test(result)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/suite.py", line 122, in run
    test(result)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/suite.py", line 122, in run
    test(result)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/case.py", line 653, in __call__
    return self.run(*args, **kwds)
  File "/home/sacastro/.pyenv/versions/3.6.7/lib/python3.6/unittest/case.py", line 624, in run
    result.stopTest(self)
  File "/snap/pycharm-professional/167/helpers/pycharm/teamcity/unittestpy.py", line 261, in stopTest
    output = sys.stdout.getvalue()
AttributeError: 'TeeLogger' object has no attribute 'getvalue'
```

Reply:
I would feel more comfortable with this change if I knew that the return value was correct.  Can you just keep this change local to your dev environment?

Reply:
Yes, that's the idea. Maybe I shouldn't have committed it.